task-manager.go

```diff
@@ -85,42 +86,44 @@ func (t *TaskManager) run() {
 				}
 			}
 		case task := <-t.InputTasksChannel:
-			// TODO hostname check is an optimisation to avoid HTTP calls to mesos slave
-			// currently in our setup hostnames a little messed up, so disabling this chck for now
-			// if task.Hostname == hostname {
-			// println("Got task for addition.. do what needs to be done")
-			// fmt.Printf("%v\n", task)
-			t.TasksMutex.Lock()
-			_, present := t.KnownTasks[task.TaskID]
-			if !present {
-				log.Printf("TaskID %s is not monitored, sending it to LogManager", task.TaskID)
-				slaveState, _ := t.Client.SlaveState(fmt.Sprintf("http://%s:%d/state.json", hostname, t.SlavePort))
-				// fmt.Printf("%v\n", slaveState)
-				executor := slaveState.FindExecutor(task.TaskID)
-				if executor != nil {
-					logFiles := strings.Split(maps.GetString(task.Labels, LogFilesToMonitor, "stdout"), ",")
-					t.KnownTasks[task.TaskID] = time.Now()
-					for _, file := range logFiles {
-						taskInfo := TaskInfo{
-							App:      task.App,
-							Hostname: task.Hostname,
-							Labels:   task.Labels,
-							TaskID:   task.TaskID,
-							CWD:      executor.Directory,
-							FileName: file,
+	//		if task.Hostname == hostname {
```

Comment:
@aayushKumarJarvis Please revert this section to the one in the master. Check the diff above.